spec/factories/questions.rb

```diff
@@ -1,9 +1,14 @@
 FactoryBot.define do
-  factory :question do
+  factory :question_same do
```

Comment:
А это что за фабрика и зачем она?

Reply:
because I need to check that question is not changed in the update method for questions_controller.
But I don't know how to check it with the dynamically generated answer

Reply:
Не понял проблемы. И я не вижу нигде в коде использования этой фабрики